dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/relationship/DefaultRelationshipService.java

```diff
@@ -63,46 +64,32 @@ public class DefaultRelationshipService implements RelationshipService {
 
   private final org.hisp.dhis.relationship.RelationshipStore relationshipStore;
 
-  private final TrackedEntityService trackedEntityService;
-
-  private final EnrollmentService enrollmentService;
-
-  private final EventService eventService;
-
   private final RelationshipOperationParamsMapper mapper;
 
   @Override
-  public List<Relationship> getRelationships(RelationshipOperationParams params)
+  public Relationships getRelationships(RelationshipOperationParams params)
       throws ForbiddenException, NotFoundException {
     RelationshipQueryParams queryParams = mapper.map(params);
 
-    if (queryParams.getEntity() == null) {
-      return List.of();
-    }
-
-    if (queryParams.getEntity() instanceof TrackedEntity te) {
-      return getRelationshipsByTrackedEntity(te, queryParams);
-    }
+    Pager pager;
+    List<Relationship> relationships = getRelationships(queryParams);
 
-    if (queryParams.getEntity() instanceof Enrollment en) {
-      return getRelationshipsByEnrollment(en, queryParams);
+    if (Boolean.TRUE.equals(queryParams.isSkipPaging())) {
```

Comment:
In case `queryParams.isSkipPaging()` is a primitive boolean it should 

```suggestion
    if (queryParams.isSkipPaging()) {
```

🤔

Reply:
It is not a primitive in https://github.com/dhis2/dhis2-core/blob/271e60c8281a045befd6ad13a224252489c5f871/dhis-2/dhis-api/src/main/java/org/hisp/dhis/webapi/controller/event/webrequest/PagingAndSortingCriteriaAdapter.java#L64